content/2013/01/01/susy-next.rst

```diff
@@ -1,5 +1,5 @@
 public: yes
-tags: [susy, "2.0", layout, grids, rwd]
+tags: [Susy, "2.0", Layout, Grids]
```

Comment:
`2.0` doesn't seem that useful, unless we attach it to the Susy tag (which I think would be a good idea). Most Susy articles will reference a specific version.